sympy/matrices/tests/test_eigen.py

```diff
@@ -142,18 +142,6 @@ def NS(e, n):
     assert isinstance(m.eigenvals(simplify=lambda x: x, multiple=True), list)
 
 
-@slow
-def test_eigen_slow():
-    # issue 15125
-    from sympy.core.function import count_ops
-    q = Symbol("q", positive = True)
-    m = Matrix([[-2, exp(-q), 1], [exp(q), -2, 1], [1, 1, -2]])
-    assert count_ops(m.eigenvals(simplify=False)) > \
-        count_ops(m.eigenvals(simplify=True))
-    assert count_ops(m.eigenvals(simplify=lambda x: x)) > \
-        count_ops(m.eigenvals(simplify=True))
```

Comment:
I'm removing this test as discussed above. The test fails here because simplify increases the op count but that is not really a useful test because it is really testing simplify rather than eigenvals and also it is not necessarily a bug in simplify (or eigenvals) if the op count increases.

There could perhaps be a better way to test for the effect of the simplify parameter but really I would prefer it if methods like `eigenvals` did not use generic `simplify` at all. The test passes on master because simplify changes `exp` into `cosh` which is considered to reduce the op count but I don't think that changing exp into cosh is something that either eigenvals or charpoly should do. It might seem that cosh is simpler than exp in the context of some particular expression but e.g. if you have other expressions involving exp then it is better to keep them all in terms of exp rather than change some to cosh.